posts/2017-03-15-faunadb-serverless-authentication.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Using Serverless Authentication Boilerplate with FaunaDB
+description: FaunaDB is a security aware database, so we pass user-specific access tokens to it via a custom authorizer.
```

Comment:
new one: "Use FaunaDB's secure database features with AWS API Gateway to run lambdas with the capabilities of the authenticated user."